docs/build

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eux
+
+docker run -p 8080:80 -v ${PWD}:/spec docker.io/redocly/cli join spec/docs.yaml spec/polaris-management-service.yml spec/rest-catalog-open-api.yaml -o spec/index.yaml --prefix-components-with-info-prop title
```

Comment:
Since we moving this, mind adding a `--rm` to the docker options?

Reply:
Agreed, added `--rm` option.